myokit/_model_api.py

```diff
@@ -2055,7 +2055,7 @@ def inits(self):
         def StateDefIterator(model):
             for k, var in enumerate(model._state):
                 yield Equation(
-                    myokit.Name(var), myokit.Number(self._current_state[k]))
+                    myokit.Name(var), self._current_state[k])
```

Comment:
Probably fits on one line now?
Esp if we switch to `for var, state in zip(model._state, self._current_state)`